lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/service/LessonBuilderAccessService.java

```diff
@@ -745,9 +789,15 @@ public void handleAccess(HttpServletRequest req, HttpServletResponse res, Refere
 							
 							// note that by default inline is always set. If we have inline and dangerous (i.e. HTML or
 							// potentially HTML, we set security headers to provide some protection.
-							if (inline && dangerous && useCsp) {
-							    res.addHeader("Content-Security-Policy", "sandbox allow-forms allow-scripts allow-top-navigation allow-popups allow-pointer-lock");
-							    res.addHeader("X-Content-Security-Policy", "sandbox allow-forms allow-scripts allow-top-navigation allow-popups allow-pointer-lock");
+
+							//This differens if it's a student page or non-student page
+							if (studentcontent && inline && dangerous && useCspStudent) {
+							    res.addHeader("Content-Security-Policy", CSPHeadersStudent);
+							    res.addHeader("X-Content-Security-Policy", CSPHeaders);
```

Comment:
Missing "Student" on the end of this variable, which allows regular CSP rules to appear on Student Pages for browsers that don't support CSP or those that favour XCSP over CSP. We'll fix this.